portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase

```diff
@@ -1296,6 +1296,87 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-133280. Poshi Automation for Bug LPS-129830 Under certain situations, unable to remove multiple manually set assets."
+	@priority = "4"
+	test ViewRemovedWebContentInAssetPublisherConfiguration {
+		task ("Go to GogoShell in System") {
```

Comment:
Maybe we can group these four steps in one, like: "Stop and start module controller portlet from GogoShell"